src/views/account/Chat/Modals/ChatCreate.tsx

```diff
@@ -174,24 +176,27 @@ const ChatCreate: Screen<"ChatCreate"> = ({ navigation }) => {
       <View style={[styles.fixedButtonContainer, {backgroundColor: colors.background}]}>
         <ButtonCta primary value={"Créer la discussion"} disabled={!(content && selectedRecipients.length > 0)} onPress={() => {
           if (!subject) {
-            Alert.alert(
-              "Veux-tu continuer sans objet ?",
-              "Tu es sur le point de créer une discussion sans objet. Veux-tu continuer ?",
-              [
+            showAlert({
+              title: "Veux-tu continuer sans objet ?",
+              message: "Tu es sur le point de créer une discussion sans objet. Veux-tu continuer ?",
+              icon: <BadgeHelp />,
+              actions: [
                 {
-                  text: "Annuler",
-                  style: "cancel",
+                  title: "Annuler",
+                  icon: <Undo2 />,
+                  primary: true,
```

Comment:
```suggestion
                  primary: false,
```

Ça fait bizarre d'avoir deux boutons rouges, comme ça ça donne mieux.